pages/en/learn/getting-started/nodejs-with-typescript.md

```diff
@@ -117,6 +117,57 @@ TypeScript offers a whole lot of other great mechanisms like interfaces, classes
 
 Some of the other benefits of TypeScript that are worth mentioning are that it can be adopted progressively, it helps making code more readable and understandable and it allows developers to use modern language features while shipping code for older Node.js versions.
 
+## Running TypeScript Code in Node.js
+
+Node.js canot run TypeScript natively. You cannot call `node example.ts` from the command line directly. But there are three solutions to this problem:
+
+### Compiling TypeScript to JavaScript
+
+If you want to run TypeScript code in Node.js, you need to compile it to JavaScript first. You can do this using the TypeScript compiler `tsc` as shown earlier.
+
+Here's a small example:
+
+```bash
+npx tsc example.ts
+node example.js
+```
+
+### Running TypeScript Code with `ts-node`
+
+You can use [ts-node](https://typestrong.org/ts-node/) to run TypeScript code directly in Node.js without the need to compile it first. But it's not typechecking your code. So we recommend to type check your code first with `tsc` and then run it with `ts-node` before shipping it.
```

Comment:
btw if you want to transpile with tsc you need to have installed typescript in the project or globally first.

maybe add that instruction as well?

```then run it with `ts-node` before shipping it.```
also when we talk about shipping the code just running ts-node does not transpile the typescript file to js it just runs. so maybe need to add more clear description on that

Reply:
I'll take a look about this.